CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
```

Comment:
I think we may need to indicate somehow that there are some known issues with relative volume paths on windows.